packages/typespec-client-generator-core/lib/decorators.tsp

```diff
@@ -6,6 +6,7 @@ namespace Azure.ClientGenerator.Core;
  * Changes the name of a method, parameter, property, or model generated in the client SDK
  * @param rename The rename you want applied to the object
  * @param scope The language scope you want this decorator to apply to. If not specified, will apply to all language emitters
+ * You can use "!" to specify negation such as "!(java, python)" or "!java, !python". Please note that, negation scope and normal scope should not be mixed.
```

Comment:
This would probably need update. Would recommend mentioning how we handle the overlapping edge case as well.

Reply:
If you are talking about how to deal with decorator override. I think that should be a separate topic.
@tadelesh Do we have any documentation about how decorator override should work?

Reply:
I mean `Please note that, negation scope and normal scope should not be mixed.` this part, may need scroll to the right a bit.. 
Though user can definitely benefit from scope override documentation.

Reply:
Repasting the description of the PR here
Combination of same scope is also allowed
```
@clientName("A", "!python, python")
Model foo
```
is equivalent to
```
@clientName("A", "!python")
@clientName("A", "python")
Model foo
```
and equivalent to
```
@clientName("A")
Model foo
```
For the case you mentioned, there is no override.

Reply:
I see. Then it's fine.

Reply:
> I mean `Please note that, negation scope and normal scope should not be mixed.` this part, since we now allow mix of negation + normal scope. May need scroll to the right a bit.. Though user can definitely benefit from scope override documentation.

remoevd